### ISSUE TITLE
Support for merge cells in tables

### ISSUE THREAD

**Issue by josecastillolema** [marp-team/marp-core#373]
First of all, thanks for the amazing tool!

I was wondering if there is a way to merge cells in a table.

If there is not, I personally found [doxygen](https://github.com/doxygen/doxygen/issues/8864) format very straighforward:
```
| Cell1| Cell2 | Cell3 |
|------|-------|-------|
| merge right ||       |
| merge|       |       |
| below|^      |^      |
| merge both  || value1|
|^            || value2|
```

**Comment by yhatt** [comment]
That syntax is not defined in [the specifications for GFM (GitHub Flavored Markdown)](https://github.github.com/gfm/#tables-extension-). There are only a few Markdown flavors that adopt that syntax, and if Marp adopts that, it could negatively impact interoperability with other Markdown processors. Therefore, this feature should be provided as a third-party plugin (e.g. [markdown-it-multimd-table](https://github.com/redbug312/markdown-it-multimd-table))

```javascript
import { Marp } from '@marp-team/marp-core';
import markdownItMultimdTable from 'markdown-it-multimd-table';

const marp = new Marp().use(markdownItMultimdTable);

marp.render(/* ... */);
```

**Comment by josecastillolema** [comment]
Thanks for the quick response.

Where do these imports should be placed? In the markdown file?

**Comment by yhatt** [comment]
See the [usage](https://github.com/marp-team/marp-core#usage) section of this repository. Marp Core is a JavaScript library so the plugin also should import in JavaScript.

Or are you using Marp Core indirectly, through other tools such as Marp CLI?

**Comment by josecastillolema** [comment]
Yeah, I am using Marp Core through the VSCode extension. In this scenario, where should I place this setup?

Thanks!

**Comment by yhatt** [comment]
VS Code does not support any plugins due to security reasons. Check out https://github.com/marp-team/marp-vscode/issues/135#issuecomment-611275071.

Whenever using plugins, you have to use [Marp CLI](https://github.com/marp-team/marp-cli#functional-engine) instead.
